content/applications/websites/livechat.rst

```diff
@@ -5,80 +5,287 @@
 Live Chat
 =========
 
-Live Chat has the highest satisfaction rating of any communication tool. It allows fast responses
-and it is accessible and convenient, as your customers can keep doing what they are doing while
-interacting with you. Remember: your customers want to talk to you, so let's make it easy.
+Odoo *Live Chat* allows users to communicate with website visitors in real time. With *Live Chat*,
+leads can be qualified for their sales potential, support questions can be answered quickly, and
+issues can be directed to the appropriate team for further investigation (or follow up). *Live Chat*
+also provides the opportunity for instant feedback from customers.
 
-.. seealso::
-   - `Live Chat: product page <https://www.odoo.com/app/live-chat>`_
-   - :doc:`livechat/ratings`
-   - :doc:`livechat/responses`
+Enable Live Chat
+================
+
+In order to enable *Live Chat*, the *Live Chat* application needs to be installed. This can be done
+in one of two ways.
+
+- Go to :menuselection:`Main Odoo Dashboard --> Apps --> Live Chat` and click :guilabel:`Install`.
```

Comment:
I believe this is writing style, so feel free to ignore this comment but I don't feel it is necessary to include **Main Odoo Dashboard** every time in the navigation between apps/modules, @StraubCreative wdyt?

Reply:
The guidelines aren't very clear and could use a long overdue update, but it's there as an example in the table. ^^'
```
:menuselection:`App name --> Menu --> Sub-menu`
```
https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#document-structure

Similarly, you would group the menu selection in the next bullet point:
```
Go to :menuselection:`Website --> Configuration --> Settings`, check the box next to :guilabel:`Livechat`,
  and click :guilabel:`Save`.
```

Also, in general, we try not to reexplain everything everywhere: it's best to use custom links or "see also" sections, such as.
```
:ref:`Install <general/install>`
```